tests/deploy_test.go

```diff
@@ -44,7 +38,7 @@ func Test_Access_Secret(t *testing.T) {
 
 		out := strings.TrimSuffix(string(bytesOut), "\n")
 		if out != secret {
-			t.Errorf("want: %q, got: %q", secret, out)
+			t.Fatalf("want: %q, got: %q", secret, out)
```

Comment:
What's the difference between fail and return vs fatal?

Does fatal call exit-1?